renpy/ast.py

```diff
@@ -2201,9 +2201,22 @@ class UserStatement(Node):
     translation_relevant: bool = False # type: ignore
     rollback: RollbackType = "normal"
     subparses: list['renpy.lexer.SubParse'] = [ ]
-    init_priority: SignedInt | None = 0
     atl: 'renpy.atl.RawBlock | None' = None
 
+    init_priority: SignedInt | None = None
+    """
+    Used to store statement init priority before 8.4.
+
+    All new instances should keep this set to None.
+    """
+
+    init_offset: SignedInt | None = 0
```

Comment:
Suggest defaulting this to `None` as it's the more common value, thus saving the most space in cslots.